src/main/java/org/lsc/jndi/JndiServices.java

```diff
@@ -1135,30 +1136,8 @@ public Map<String, LscDatasets> doGetAttrsList(final String base,
 		constraints.setReturningAttributes(attributes);
 		constraints.setSearchScope(scope);
 		constraints.setReturningObjFlag(true);
-
```

Comment:
Same comment as previous:

In this function, you should:
* call a new function setsearchcontrols() before the search. This function would set the pagesize and sortedby controls
* call a new function clearsearchcontrols() after the search. This function would get the current controls (with  getRequestControls), and remove only the paged and sortedby controls if they exist. And also take care to call the clearsearchcontrols in the catch clauses.